Tests/ToolBox/LeanDataReaderTests.cs

```diff
@@ -221,5 +222,162 @@ private void ConvertMinuteFuturesData(Symbol canonical, TickType tickType, Resol
         }
 
         #endregion
+
+
+        [Test, TestCaseSource(nameof(OptionAndFuturesCases)), Category("TravisExclude")]
+        public void ReadLeanFutureAndOptionDataFromFilePath(string composedFilePath, Symbol symbol,  int rowsInfile, double sumValue)
+        {
+            // Act
+            var ldr = new LeanDataReader(composedFilePath);
+            var data = ldr.Parse().ToArray();
+            // Assert
+            Assert.True(symbol.Equals(data.First().Symbol));
+            Assert.AreEqual(rowsInfile, data.Length);
+            Assert.AreEqual(sumValue, data.Sum(c => c.Value));
+        }
+
+
+        public static object[] OptionAndFuturesCases =
+        {
+            new object[]
```

Comment:
no change required -- it's nice to use the `TestCaseData` class because then you can use `SetName` to give the test case a human readable name in the test window